Coding/fw/src/Adapters/NVMAdapter.h

```diff
@@ -16,7 +16,7 @@ ESP32 NVM Adapter for Lawicel Protocol
 
 /* INCLUDES ***************************************************************************************/
 #include <NVMInterface.h>
-#include <Preferences.h>
+#include <Settings.h>
```

Comment:
You need to fix all your includes. Only system includes use <>-bracker notation.